frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -149,7 +142,7 @@ const FileHandler = ({
         event.preventDefault();
 
         if (fileContent.length === 0) {
-            showError(`No File Contents To ${submitText}`);
+            showError(`No file contents To validate`);
```

Comment:
nit: doesn't need to be be a template literal since there's no interpolation happening.

Reply:
Good catch -- will update!